components/gitpod-db/migrate-migrations/0_2_0_down_procedure.sql

```diff
@@ -0,0 +1,21 @@
+-- Copyright (c) 2021 Gitpod GmbH. All rights reserved.
```

Comment:
We never call this. Is that on purpose?

Reply:
Yes. It has been useful for testing, and might serve well in case something goes south during deployment, that's why I left it here.